pkgs/development/interpreters/apeloader-bin/default.nix

```diff
@@ -0,0 +1,41 @@
+{ stdenvNoCC, lib, fetchurl, cosmopolitan, nixosTests }:
+
+stdenvNoCC.mkDerivation {
+  pname = "apeloader-bin";
+  version = "1.o";
```

Comment:
is it really 1.o or 1.0 ?

Reply:
It is `1.o` in the source material. I am not sure exactly how to treat it however, it seems to be a pun on `1.0`.

Reply:
Hm, tough question, obviously, it means 1.0, but if we do want to compare versions, 1.o and 1.1 are not easily comparable. I have no hard feelings on that, I would rather use 1.0 and convert the puns, but it might break tools such as ryan-tm's bot.